great_expectations/expectations/metrics/table_metrics/table_columns.py

```diff
@@ -29,7 +29,7 @@ def _pandas(
         execution_engine: PandasExecutionEngine,
         metric_domain_kwargs: Dict,
         metric_value_kwargs: Dict,
-        metrics: Dict[Tuple, Any],
+        metrics: Dict[Union[Tuple, str], Any],
```

Comment:
@jcampbell I suspect that there is a bug upstream.  This is because with just `metrics: Dict[Tuple, Any],` (instead of my correction of `metrics: Dict[Union[Tuple, str], Any],`, the next-to-last line below (`column_metadata = metrics["table.column_types"]`) generates a PyCharm warning.
```
    @metric_value(engine=PandasExecutionEngine)
    def _pandas(
        cls,
        execution_engine: PandasExecutionEngine,
        metric_domain_kwargs: Dict,
        metric_value_kwargs: Dict,
        metrics: Dict[Union[Tuple, str], Any],
        runtime_configuration: Dict,
    ):
        column_metadata = metrics["table.column_types"]
        return [col["name"] for col in column_metadata]
```
The warning message is:
```
Expected type 'tuple' (matched generic type '_KT'), got 'str' instead 
```
Should I undo my "fix" so that we can come up with a true fix in the place that does NOT maintain the tuple as the key?  Or should we keep it?  Thanks!